src/target/riscv/riscv.c

```diff
@@ -5287,6 +5311,14 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "[('eq'|'napot'|'ge_lt'|'all') ('wp'|'none')]",
 		.help = "Control whether OpenOCD is allowed to use certain RISC-V trigger features for watchpoints."
 	},
+	{
+		.name = "reserve_trigger",
+		.handler = handle_reserve_trigger,
+		/* TODO: Move this to COMMAND_ANY */
+		.mode = COMMAND_EXEC,
+		.usage = "[index [('on'|'off')]]",
+		.help = "Controlls which triggers can be used by OpenOCD.",
```

Comment:
I've fixed a typo ("controls") and tried to reword the sentence to make it even more clear.

```suggestion
		.help = "Controls which RISC-V triggers shall not be used (not touched anyhow) by OpenOCD.",
```

Reply:
Addressed. Changed to make a bit shorter. Please, take a look.

Reply:
Looks good and very clear - thanks.